Abstractinterface.java

```diff
@@ -1,4 +1,4 @@
-
+//抽象インターフェース
 public abstract class Abstractinterface {
```

Comment:
write with upper camel case; Abstract Bombed ExpoInterface

Reply:
thank you for rewriting it!!